src/ConnectFour/Domain/Game/Board/Size.php

```diff
@@ -8,32 +8,29 @@
 
 final class Size
 {
-    private int $width;
-
-    private int $height;
-
     /**
      * @throws InvalidSizeException
      */
-    public function __construct(int $width, int $height)
-    {
+    public function __construct(
+        public readonly int $width,
+        public readonly int $height
+    ) {
         if ($width < 2 || $height < 2) {
             throw new InvalidSizeException('Width and height must be greater then 1.');
         }
-
-        if (($width * $height) % 2 !== 0) {
```

Comment:
That logic made sense because otherwise player 1 would have more moves than player 2. However, a 7x9 board size is valid according to Wikipedia.